internal/daemon/daemon.go

```diff
@@ -635,10 +635,6 @@ func (d *Daemon) Stop(sigCh chan<- os.Signal) error {
 	return nil
 }
 
-// This is a little more than servstate.killWait (but that isn't and shouldn't
-// be exported, so just duplicate it here).
-const stopRunningTimeout = 5*time.Second + 100*time.Millisecond
-
```

Comment:
Do we still want to remove this? If so, why?

Reply:
Because it triggers well before the service is stopped and then forces the reaper to be stopped. Doesn't make a huge amount of sense in my opinion.

Reply:
Try that out based on our discussions.